player.rb

```diff
@@ -0,0 +1,30 @@
+# Class for a blackjack player to keep track of cards in hand and total value
+class Player
+  attr_reader :hand, :total
+  attr_accessor :hand, :total
+
+  def initialize
+    @hand = []
+    @total = calculate_total
+  end
+
+  def get_card_val(sum, card)
```

Comment:
I think this method represents another mixing of concerns: the value of a card and the total value of a player's hand. It's true that whether an ace is worth 1 or 11 depends on knowledge of the broader context, but it's not true that the value of a jack, queen, king or number card does. I might leave the number logic to the card itself, and then account for aces in a method that determines the hand's value here.